tests/logger.rs

```diff
@@ -13,24 +16,32 @@ mod test_module {
     #[test]
     fn test_logger() {
         // Line number of tracing::info! location
-        const INFO_LINE: u32 = 51;
+        const INFO_LINE: u32 = 63;
 
         let call_count = Arc::new(AtomicUsize::new(0));
 
         #[derive(Debug)]
         struct TestLogger {
             call_count: Arc<AtomicUsize>,
+            log_caller_tid: ThreadId,
         }
         impl TelioLoggerCb for TestLogger {
             fn log(
                 &self,
                 log_level: telio::ffi_types::TelioLogLevel,
                 payload: String,
             ) -> FfiResult<()> {
+                if payload == "Starting async logger thread" {
+                    return Ok(());
+                }
```

Comment:
Nit: this is detached from the actual message, such upon change, this conditional will break. It poses no harm but will eventually become a question "what is this log line in my logs". I suggest making a constant of it or just removing the check altogether :)